firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/HAL/Endian.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * Endian.h
+ *
+ *  Created on: May 24, 2020
+ *      Author: Raavi Lagoo & Renji Panicker
+ *
+ *  Endianness conversions between host and network order.
+ *  STM32 is little-endian, while network order is big-endian, so
+ *  the conversion functions reverse the order of bytes..
+ */
+
+#pragma once
+
+#include "Pufferfish/HAL/STM32/Endian.h"
+
+namespace Pufferfish {
+namespace HAL {
+
+template <typename T>
+constexpr T hton(const T& i) {
+  return Pufferfish::HAL::STM32::hton(i);
```

Comment:
Let's use `using Platform = Pufferfish::HAL::STM32`, and then we can use `return Platform::hton` and `return Platform::ntoh`.

Reply:
namespace Platform = Pufferfish::HAL::STM32 works well. I have updated this change.